### PR TITLE
Vue front dev

### PR DESCRIPTION
**Pull request by kompilorb**
Pull Requet modification ticket GS vue (filtre)

### CODE REVIEW

File: admin-dev/themes/new-theme/js/app/widgets/ps-radio.vue [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        value: { type: String, required: false },
+    },
+    methods: {
+        onChange() { this.$emit('change', this.value); },
+    },
+}
+</script>
+
+<style>
+</style>
```
Comment (maximebiloe) [inline admin-dev/themes/new-theme/js/app/widgets/ps-radio.vue]: Is this really necessary the `<style>` tag?

File: admin-dev/themes/new-theme/js/app/widgets/ps-radio.vue [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2017 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *-->
+
+<template>
+    <div class="radio">
```
Comment (maximebiloe) [inline admin-dev/themes/new-theme/js/app/widgets/ps-radio.vue]: The indent for vue files is two spaces

File: admin-dev/themes/new-theme/js/app/pages/stock/components/header/filters.vue [hunk 3]
```diff
@@ -76,6 +76,29 @@
             />
           </div>
         </div>
+        <div class="col-md-4">
+          <div class="p-y-2 p-x-2">
+            <h2>{{trans('filter_status')}}</h2>
+            <PSRadio
+                :label="trans('filter_status_enable')"
```
Comment (maximebiloe) [inline admin-dev/themes/new-theme/js/app/pages/stock/components/header/filters.vue]: The indent is 2 spaces, not 4

File: admin-dev/themes/new-theme/js/app/widgets/ps-radio.vue (hunk 2 above)
Comment (nihco2) [inline admin-dev/themes/new-theme/js/app/widgets/ps-radio.vue]: I have to review the files with my own eslint config

File: admin-dev/themes/new-theme/js/app/widgets/ps-radio.vue (hunk 1 above)
Comment (nihco2) [inline admin-dev/themes/new-theme/js/app/widgets/ps-radio.vue]: Yes, I will add styles this morning

File: admin-dev/themes/new-theme/js/app/pages/stock/components/header/filters.vue (hunk 3 above)
Comment (nihco2) [inline admin-dev/themes/new-theme/js/app/pages/stock/components/header/filters.vue]: Same comment as before ;)

File: admin-dev/themes/new-theme/js/app/widgets/ps-radio.vue (hunk 1 above)
Comment (maximebiloe) [inline admin-dev/themes/new-theme/js/app/widgets/ps-radio.vue]: So, its not ready to be merged

File: admin-dev/themes/new-theme/js/app/widgets/ps-radio.vue (hunk 2 above)
Comment (maximebiloe) [inline admin-dev/themes/new-theme/js/app/widgets/ps-radio.vue]: There is a .editorconfig in the repo, just install the plugin 😉

**Pull request closed without merging**